src/docker/10-debianconsole/.prebuild.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# FIXME Remove this when Docker supplies multi-arch base images
+if [ "$ARCH" == "arm" ]; then
+  BASE_IMAGE=debian:jessie
+  ARM_BASE_IMAGE=armhf/debian:jessie
```

Comment:
I've tested it and `armhf/debian` doesn't work on RPi 1. `resin/rpi-raspbian` does.
I don't know why `armhf/*` images fail on RPi 1 when it's `armhf`, but I think it has something to do with `VFPv3` support or something (compare `GOARM=6` and `GOARM=7`)